distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/csw/mock/FederatedCswMockServer.java

```diff
@@ -221,4 +213,13 @@ private String substituteSourceId(String document) {
     private String substituteHttpRoot(String document) {
         return document.replaceAll("\\$httpRoot\\$", httpRoot);
     }
+
+    private String getDefaultResponseResource(String resourceName) {
+        try {
+            return IOUtils.toString(FederatedCswMockServer.class.getResourceAsStream(
```

Comment:
I wanted to make this resource loading specific to this class since it is standalone so I avoided using get `getFileContent` which would coupled it to AbstractIntegrationTest. I should definitely be using `getClassLoader().getResourceasStream(resourceName)` though, will change :+1: